File: src/main/java/com/example/board/dto/BoardDto.java
package com.example.board.dto;
import lombok.NoArgsConstructor;
import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
public class BoardDto {
    private int id;
    private String boardTitle;   // 게시판 제목
    private String boardContent; // 게시판 내용
    private String userNo;       // 사용자 번호
    private LocalDateTime createDate;
    private int delYn;
}

File: src/main/kotlin/com/example/board/dto/BoardDto.kt
package com.example.board.dto

import java.time.LocalDateTime

data class BoardDto (
    var id: Int = 0,
    var boardTitle : String? = "",   // 게시판 제목
    var boardContent :  String ?= "", // 게시판 내용
    var userNo : String ?= "",      // 사용자 번호
    var createDate : LocalDateTime ?= LocalDateTime.now(),
    var delYn : Int = 0
)
